app/routes/device.py

```diff
@@ -169,7 +182,7 @@ def ser():
     serialInst.open()
 
     while True:
-        socketio.sleep(0)
+        socketio.sleep(5000)
```

Comment:
We had to set a positive integer (> 0) for this not to freeze, before
Does that no longer happen?